lib/membrane/rtp/outbound_tracking_serializer.ex

```diff
@@ -13,6 +13,8 @@ defmodule Membrane.RTP.OutboundTrackingSerializer do
   alias Membrane.RTP.Session.SenderReport
   alias Membrane.{Buffer, Payload, RemoteStream, RTCPEvent, RTP, Time}
 
+  @padding_packet_size 256
```

Comment:
I think this should be 255 as the max padding size is denoted by last octet of RTP packet payload

Reply:
It's not a big probem, the header will certainly be bigger than 1 byte, but sure, this would make more sense. We could also consider using 255 + min header size

Reply:
I missunderstood the code at the first time. I thought that `@padding_packet_size` is the length of RTP packet payload. In such a case, 256 would result in 0 in last byte denoting padding size as we would have an overflow.

Now I see that we never get padding packet that has 255 zeros. Instead we always get packet that is 255 bytes long in total, including header.

I think we should fix this. If we want to create padding packet it should have 255 bytes + 1 byte for padding size + header

Reply:
> Padding packet should have 255 + 1 byte for padding size

Nitpick: 255 bytes of padding, period. Padding size is part of padding